### PR TITLE
Swaps: coin row info button

### PR DESCRIPTION
**Pull request by benisgold**
Fixes APP-1527
## Screen recordings / screenshots

https://github.com/rainbow-me/rainbow/assets/15272675/5c5e2710-4c13-4b1a-bc2f-182558cc9427

https://github.com/rainbow-me/rainbow/assets/15272675/b04231a5-e7dc-4075-a08c-e21a7092b77c

### CODE REVIEW

File: src/__swaps__/screens/Swap/components/CoinRow.tsx [hunk 1]
```diff
@@ -11,7 +11,104 @@ import Animated from 'react-native-reanimated';
...
+              },
+            },
+          ]
+        : []),
+    ],
+    menuTitle: '',
+  };
+
+  const handlePressMenuItem =
+    // @ts-expect-error ContextMenu is an untyped JS component and can't type its onPress handler properly
```
Comment (walmat) [inline src/__swaps__/screens/Swap/components/CoinRow.tsx:72]: I see this multiple areas in our app and more of a nitpick, but the proper type is `OnPressMenuItemEventObject`.

File: src/__swaps__/screens/Swap/components/SearchInput.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ export const SearchInput = ({
     return {
       defaultValue,
       text: query,
-      selectionColor: getColorValueForThemeWorklet(asset.value?.highContrastColor, isDarkMode, true),
+      // selectionColor: getColorValueForThemeWorklet(asset.value?.highContrastColor, isDarkMode, true),
```
Comment (walmat) [inline src/__swaps__/screens/Swap/components/SearchInput.tsx:58]: revert before merge

File: src/__swaps__/screens/Swap/components/CoinRow.tsx [hunk 3]
```diff
@@ -11,7 +11,104 @@ import Animated from 'react-native-reanimated';
...
+
+  const options = {
+    copy: {
+      title: i18n.t(i18n.l.exchange.coin_row.copy_contract_address),
+      action: handleCopy,
+    },
+    ...(network
+      ? {
+          blockExplorer: {
+            title: i18n.t(i18n.l.exchange.coin_row.view_on, { blockExplorerName: startCase(ethereumUtils.getBlockExplorer(network)) }),
```
Comment (walmat) [inline src/__swaps__/screens/Swap/components/CoinRow.tsx:38]: i don't think you're consuming the `blockExplorerName` in the i18n entry.

Comment (benisgold) [reply src/__swaps__/screens/Swap/components/CoinRow.tsx:38]: this is correct right?

Screenshot 2024-05-31 at 9 14 28 AM

Comment (walmat) [reply src/__swaps__/screens/Swap/components/CoinRow.tsx:36]: oh weird the git diff didn't show that, my bad

**Review by walmat — APPROVED** [review]

**Pull request merged**